src/ddsrt/tests/hopscotch.c

```diff
@@ -237,11 +237,10 @@ static void chhtest_gc (void *block, void *arg)
 struct chhtest_thread_arg {
   ddsrt_atomic_uint32_t *stop;
   struct ddsrt_chh *chh;
-  uint32_t nkeys;
   uint32_t *keys;
-  bool check;
-
+  uint32_t nkeys;
```

Comment:
clang-tidy suggested it. rather than disabling the check, I thought the type of suggestion may come in handy in places where it does matter and so I opted to change test